templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -751,18 +749,30 @@ contents:
     # Setup an exit trap to rollback on error
     handle_exit() {
       e=$?
-      [ $e -eq 0 ] && print_state && exit 0
+      tdir=$(mktemp -u -d -t "configure-ovs-$(date +%Y-%m-%d-%H-%M-%S)-XXXXXXXXXX")
+      
+      if [ $e -eq 0 ]; then
+        print_state
+        # remove previous troubleshooting information
+        rm -rf "$(dirname "$tdir")/configure-ovs-*"
+        exit 0
+      fi
 
       echo "ERROR: configure-ovs exited with error: $e"
       print_state
 
-      # copy configuration to tmp
-      dir=$(mktemp -d -t "configure-ovs-$(date +%Y-%m-%d-%H-%M-%S)-XXXXXXXXXX")
+      # remove previous troubleshooting information except the oldest one
+      mapfile -t tdirs < <(compgen -G "$(dirname "$tdir")/configure-ovs-*)")
+      unset -v "tdirs[0]"
+      for dir in "${tdirs[@]}"; do rm -rf "$dir"; done
+
+      # copy configuration to tmp for troubleshooting
+      mkdir -p "$tdir"
```

Comment:
nit: already created by `mktemp`.

Reply:
dam - thats why i like when folks expand arguments to binaries and make them verbose. Not everyone is familiar with the short arguments.